cli/attribute/attribute.go

```diff
@@ -48,3 +52,69 @@ func List() {
 	}
 	fmt.Printf("%s\n", jsonBytes)
 }
+
+func AddFile(org string, credToUse string, inputFile string) {
```

Comment:
This may not be the way we want user add UserInput to node.  Please remove it but save the file somewhere just in case.